server/src/main/java/io/spine/server/ContextSpec.java

```diff
@@ -38,32 +40,34 @@
 
     private final BoundedContextName name;
     private final boolean multitenant;
+    private final boolean storeEvents;
 
-    private ContextSpec(BoundedContextName name, boolean multitenant) {
+    private ContextSpec(BoundedContextName name, boolean multitenant, boolean storeEvents) {
```

Comment:
The `storeEvents` parameter looks weird and not native for a regular `BoundedContext`. What a context does not store events? It can be equally named `isSystem`... :) This looks like the idea of context hierarchy and its properties leaked into here.